Makefile

```diff
@@ -1,59 +1,95 @@
-RELEASE_DIR		= release/prototype
-DIAZO_DIR       = src/ploneintranet/theme/static
-LATEST          = $(shell cat prototype/LATEST)
+RELEASE_DIR	= release/prototype/_site
+DIAZO_DIR       = src/ploneintranet/theme/static/generated
+LATEST          = $(shell cat LATEST)
 BUNDLENAME      = ploneintranet
-BUNDLEURL		= https://products.syslab.com/packages/$(BUNDLENAME)/$(LATEST)/$(BUNDLENAME)-$(LATEST).tar.gz
+BUNDLEURL	= https://products.syslab.com/packages/$(BUNDLENAME)/$(LATEST)/$(BUNDLENAME)-$(LATEST).tar.gz
 
-all:: diazo
+all:: fetchrelease
 default: all
 clean:
 	rm bin/* .installed.cfg || true
 check-clean:
 	test -z "$(shell git status --porcelain)" || (git status && echo && echo "Workdir not clean." && false) && echo "Workdir clean."
 
+fetchrelease:
+	# update LATEST in case we updated the prototype
+	$(eval LATEST := $(shell cat LATEST))
+	$(eval BUNDLEURL := https://products.syslab.com/packages/$(BUNDLENAME)/$(LATEST)/$(BUNDLENAME)-$(LATEST).tar.gz)
+	echo $(BUNDLEURL)
```

Comment:
I'm now sure what you mean with `# update LATEST in case we updated the prototype`, looks to me like you're just reading the contents of LATEST, not checking what LATEST is in the prototype.

Also, looks to me like we would manually have to update the LATEST file in ploneintranet if we have done `make jsrelease` in the prototype, correct?